kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -1163,6 +1163,7 @@ define([
                                 this.startLogAutoFetch();
                                 break;
                             case 'finished':
+                                this.requestJobLog(0);
```

Comment:
the log viewer was missing the final call to fetch all logs; this fixes that -- see https://kbase-jira.atlassian.net/browse/DATAUP-628